pkg/volume/fc/fc_util.go

```diff
@@ -101,15 +140,28 @@ func scsiHostRescan(io ioHandler) {
 	}
 }
 
+// check scsi disk is really exist and is attached to the node using SCSI Inquiry
+func checkDiskExist(b fcDiskMounter, disk, wwid string) (bool, error) {
+	out, err := b.plugin.execCommand("/lib/udev/scsi_id", []string{"--replace-whitespace", "--page=0x83", "--whitelisted", "--device=" + disk})
```

Comment:
I am not quite worried if additional wwid check is missing. The persistent naming should already be part of the OS rules. If the code reaches here, the path is already created by udev rules and double checking the paths is redundant.